alcs-frontend/src/app/features/board/board.component.ts

```diff
@@ -31,7 +34,26 @@ export class BoardComponent implements OnInit {
     private router: Router
   ) {}
 
-  async ngOnInit() {
+  ngOnInit() {
+    this.activatedRoute.params.subscribe((params) => {
+      const decisionMakerCode = params['dmCode'];
+      if (decisionMakerCode) {
+        this.decisionMakerCode = decisionMakerCode;
+        this.applicationService.setDecisionMaker(decisionMakerCode);
+        this.applicationService.fetchApplications();
+      }
+    });
+
+    this.applicationService.$applicationDecisionMakers.subscribe((dms) => {
+      this.decisionMakers = dms;
+      const decisionMaker = dms.find((dm) => dm.code === this.decisionMakerCode);
+      if (decisionMaker) {
+        this.boardTitle = decisionMaker.label;
+      } else {
+        this.boardTitle = 'All Applications';
+      }
```

Comment:
in this case, I would prefer ternary condition or just set the `this.boardTitle='All Applications'` by default in declaration

Reply:
Good point! I moved it to initialize as All Applications instead.